client/src/utils/apiWrapper.js

```diff
@@ -39,3 +39,18 @@ export const addSampleResponse = (body) => {
       error,
     }));
 };
+
+//calling backend endpoint to edit user class in profile
+export const putUserClass = (body) => {
```

Comment:
give this a more descriptive name. putUserClass isn't intuitive on the frontend. change to `editUserClasses` or something similar